Map/Continent.h

```diff
@@ -0,0 +1,27 @@
+#include <iostream>
+using namespace std;
+#include <vector>
+#include "./Territory.h"
+
+#pragma once
+
+class Continent {
```

Comment:
The Continent class doesn’t have a destructor. It has a vector of pointers so I think it needs a destructor. The destructor would have to be defined in Map.cpp of course

Reply:
Good catch! I have removed this file since it has been merged with the Map.h file